source/docs/faq/github_account.md

```diff
@@ -3,10 +3,11 @@ id: github_account
 title: "Why do I need a GitHub account?"
 description: "Information about why a GitHub account is needed"
 ---
-HACS uses the [GitHub API](https://docs.github.com/en/rest) to gather information about all [available](https://github.com/hacs/default) and downloaded repositories. [This API is rate limited](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting) to 60 requests every hour for unauthenticated requests, which is not enough. So HACS needs to make authenticated requests to that API.
 
-A OAuth token with no scopes which grants read-only access to public information (including user profile info, repository info, and gists), will be used.
+HACS uses the [GitHub API](https://docs.github.com/en/rest) to gather information about all [available](https://github.com/hacs/default) and downloaded repositories. Unauthenticated requests on this API are [rate limited](https:/docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting) to 60 requests every hour. Because this is not enough, HACS needs to make authenticated requests. The rate limit for authenticated requests is much higher.
```

Comment:
```suggestion
HACS uses the [GitHub API](https://docs.github.com/en/rest) to gather information about all [available](https://github.com/hacs/default) and downloaded repositories. Unauthenticated requests on this API are [rate limited](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting) to 60 requests every hour. Because this is not enough, HACS needs to make authenticated requests. The rate limit for authenticated requests is much higher.
```